dts/bindings/interrupt-controller/irq.h

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
As these are IOAPIC/MVIC specific, this file should be:

intel-mvic.h & intel-ioapic.h (we can just have intel-mvic.h include intel-ioapic.h)